package.json

```diff
@@ -19,11 +19,16 @@
     "lint:ci": "eslint -f eslint-formatter-multiple src",
     "lint:all": "yarn lint:ci && yarn --cwd frontend lint:ci",
     "clean": "rm -rf coverage reports frontend/coverage frontend/reports frontend/build",
+    "api:test": "yarn docs:build && npx dredd --server 'yarn server'",
```

Comment:
```suggestion
    "api:test": "yarn docs:build && npx dredd 'yarn server'",
```

Reply:
Without `--server` `api:test` works for me locally.

Reply:
I'm having a hard time getting this one to play nicely with running on a windows and linux host and at the same time being able to be ran from docker. Maybe Dredd is more trouble then it is worth at this point? I was on the edge on if I thought documentation testing was worth it and I'm leaning more towards thinking it isn't now.

Reply:
Assuming `npx dredd 'yarn server'` doesn't work for you right?

It is nice that it tests the documentation, but we could remove it if you have second thoughts about it and possibly include it again later if needed.

Reply:
One last thing to try... can you tell me if `npx dredd --config ./dredd.yml` works for you?